tox.ini

```diff
@@ -12,6 +12,7 @@ envlist =
 minversion = 3.18
 
 [testenv]
+basepython = python3.10
```

Comment:
Why do you need to force _python3.10_? Doesn't _python3_ suffice?

Reply:
yes, python3 is sufficient. I can change it.